plugins/cilium-cni/04-flannel-cilium-cni.conflist

```diff
@@ -17,7 +17,8 @@
     },
     {
        "name": "cilium",
-       "type": "cilium-cni"
+       "type": "cilium-cni",
+       "disable-debug": true
```

Comment:
We have `debug` as an option for `cilium-agent`, while this is `disable-debug` - invert the logic and make this `debug` for consistency. Are you doing it this way because we have had debugging enabled by default?

Reply:
correct